lib/Extension/LanguageServerPhpCodeSniffer/LanguageServerPhpCodeSnifferExtension.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerPhpCodeSniffer;
```

Comment:
Let's just call this `Extension\PhpCodeSniffer` and remove the `LanguageServer` prefix. In hindsight the other extensions shouldn't have the `LanguageServer` prefix either (these extensions should encapsulate all functionality for the integration, not just for "language server" even if in practice that's what they do)